cuml/bench.py

```diff
@@ -8,6 +8,18 @@
 import sklearn
 import timeit
 import json
+import os
+import sys
+
+
+if 'FORCE_DAAL4PY_SKLEARN' in os.environ.keys():
```

Comment:
Similar to sklearn patch
```suggestion
if os.environ.get('FORCE_DAAL4PY_SKLEARN', False) in [True, '1', 'y', 'yes', 'Y', 'YES', 'Yes', 'true', 'True', 'TRUE']:
```

Reply:
Added